codec/common/inc/cpu_core.h

```diff
@@ -86,6 +86,7 @@
 
 /* For loongson */
 #define WELS_CPU_MMI        0x00000001  /* mmi */
+#define WELS_CPU_MSA        0x00000002  /* msa */
```

Comment:
can I know the reason that why you define MMI as 0x01 and MSA 0x02?

Reply:
cpu feature flags, like mmx, mmx-ext, sse...

Reply:
I know this is a cpu feature flag. I want to know the reason why you define this flag as value "1" and "2" but not other value. is there any special reason?

Reply:
no special reason